gadget/main.c

```diff
@@ -114,8 +114,10 @@ int main(int argc, char **argv)
         case 3:
             begrun(RestartSnapNum);
             /*FoF needs a tree*/
-            force_tree_rebuild();
-            fof_fof();
+            struct OctTree Tree = {0};
```

Comment:
Since the tree is slightly augmented, I suggest the name ForceTree than the more generic OctTree. It also agrees with the function and module names better.

Reply:
For sure! Actually, how about struct force_tree, since we also have struct particle_data?

Reply:
No, it shall be ForceTree to match TreeWalk